### PR TITLE
For #21313: Renewing performance awesomebar metrics

### PR DESCRIPTION
**Pull request by eliserichards**
For #21313

**Needs data review before landing**

Renewing the following metrics: 

`perf.awesomebar`:
  * history_suggestions
  * bookmark_suggestions
  * search_engine_suggestions
  * session_suggestions
  * synced_tabs_suggestions
  * shortcuts_suggestions
  * clipboard_suggestions

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by eliserichards** [comment]
# Request for Data Collection Renewal

`progressive_web_app`: homescreen_tap, install_tap, foreground, background
  
1) Provide a link to the initial Data Collection Review Request for this collection.
      - https://github.com/mozilla-mobile/fenix/pull/11859

2) When will this collection now expire?

      - All probes that were set to expire in November 2021 will now expire in November 2022. 

3) Why was the initial period of collection insufficient?
      - We are continuing to monitor this data through the MR2 release.

**Comment by eliserichards** [comment]
Tagging you for data review @travis79 😄

**Comment by travis79** [comment]
> Tagging you for data review @travis79 😄

Probably have time tomorrow or Friday, if later than that I'll see if someone else can get to it sooner. Thanks!

**Comment by travis79** [comment]
> # Request for Data Collection Renewal
> 
> `perf.awesomebar`:
> * `history_suggestions`
> * `bookmark_suggestions`
> * `search_engine_suggestions`
> * `session_suggestions`
> * `synced_tabs_suggestions`
> * `shortcuts_suggestions`
> * `clipboard_suggestions`
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [Performance telemetry for awesomebar suggestion provider durations #10276 (review)](https://github.com/mozilla-mobile/fenix/pull/10276#pullrequestreview-411101979)
> 
>     2. When will this collection now expire?
>        
>        * All probes that were set to expire in November 2021 will now expire in November 2022.
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * These metrics are used to track performance regressions in the awesomebar, so we want to keep them around.

# Data Collection Renewal Review 

1) Is the provided Data Collection Review complete, correct, and data-review+ by a Data Steward?

Yes

2) Is the data collection covered by the existing Firefox Privacy Notice?

Yes

## Result

data-reveiw+

**Review by travis79 — APPROVED** [review]
data-review+

File: app/metrics.yaml [hunk 1]
```diff
@@ -4102,13 +4102,14 @@ perf.awesomebar:
     data_reviews:
       - https://github.com/mozilla-mobile/fenix/pull/10276#pullrequestreview-411101979
       - https://github.com/mozilla-mobile/fenix/pull/19924#issuecomment-861423789
+      - https://github.com/mozilla-mobile/fenix/pull/TBD
```
Comment (grigoryk) [inline app/metrics.yaml:4105]: https://github.com/mozilla-mobile/fenix/pull/21315#issuecomment-920848442

**Review by grigoryk — APPROVED** [review]
Thank you, @eliserichards !

**Comment by grigoryk** [comment]
Please also check-in the updated`md` files, I think running a build should produce the necessary diff.

**Comment by eliserichards** [comment]
The `.md` file is no longer being generated here, see [this](https://github.com/mozilla-mobile/fenix/commit/f180119774122bac5314fa00ee1b1f4b30a4d736) 👍

**Comment by eliserichards** [comment]
Flaky test, the same one I'm seeing in [this PR](https://github.com/mozilla-mobile/fenix/pull/21342#issuecomment-928273364)
* TitleHeaderBindingTest. `WHEN grouped tabs are added to the list THEN return false`

**Comment by eliserichards** [comment]
@mergifyio rebase

**Comment by eliserichards** [comment]
More tests failing: 
- BrowserStateTest. GIVEN a tab group exists WHEN recentTabs is called THEN return a tab group
- BrowserStateTest. GIVEN the selected tab is a normal tab and tab group exists WHEN asRecentTabs is called THEN return a list of these tabs

**Pull request merged**